content/es/UI.json

```diff
@@ -1,9 +1,15 @@
 {
   "UI": {
     "Language": "Español",
+    "Flag": "🇲🇽",
```

Comment:
Also not the Spanish flag?

Reply:
Yep, I added these flags because of the part in "Reservations" about how the flags have to render to show what "Languages are available"